phala-cloud/getting-started/overview.mdx

```diff
@@ -1,28 +1,50 @@
 ---
-description: Quick start guide for deploying confidential applications on Phala Cloud.
 title: Quick Start
 ---
 
-Phala Cloud offers a secure and private hosting in a TEE using [Dstack](/dstack/overview). This guide will get you started on Phala Cloud and deploy your first CVM in minutes!
+Phala Cloud provides secure, privacy-preserving application hosting through TEE technology. This guide will walk you through the essential steps to get started with Phala Cloud and deploy your first CVM in just a few minutes.
+
+## Getting Started
+
+What you'll accomplish:
+- Establish a secure Phala Cloud account
+- Deploy your first confidential computing environment
+- Experience TEE-powered privacy protection
+- Access enterprise-grade security features
+
+Follow these steps to set up your Phala Cloud application:
 
 <Steps>
 <Step>
 ### [Create Your Phala Cloud Account](/phala-cloud/getting-started/sign-up-for-cloud-account) (1 minute)
 
-Go to the [sign-up](https://cloud.phala.network/register) page for Cloud account section to get an official Phala Cloud account to get started.
+Begin by creating your official Phala Cloud account. Visit our [registration page](https://cloud.phala.network/register) to establish your secure cloud environment and gain access to our TEE-powered infrastructure.
 </Step>
 
 <Step>
-### [Your First CVM Deployment](/phala-cloud/getting-started/start-from-cloud-ui) (2-3 minutes)
+### [Deploy Your First CVM](/phala-cloud/getting-started/start-from-cloud-ui) (2-3 minutes)
 
-Once your account is created, deploy your first CVM without any coding by deploying a Jupyter Notebook.
+After account creation, deploy your inaugural CVM. Start with a pre-configured Jupyter Notebook environment to experience the power of confidential computing without writing any code.
```

Comment:
inaugural <-- tone is too formal, and not good as a tutorial doc

In general, the tone is like enterprise sales, not developer focused